index.html

```diff
@@ -17,27 +17,14 @@
         <header>
             <nav class="nav__header">
                 <a class="logo" href="#"><img src="assets/images/logo-dark.png" alt="logo" /></a>
-                <nav class="hamburger" onclick="menuFunction()"><img src="assets/images/Union.svg" alt="menu"></nav>
-
-                <!-- Desktop Menu     -->
-
+                <nav class="hamburger"><img src="assets/images/Union.svg" alt="menu"></nav>

```

Comment:
The app is NOT performing as expected.
Please ensure that: 

- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
- When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears.

At the moment, when I click on the hamburger button, the mobile menu did not appear.

Kindly fix the issue.